src/phpDebug.ts

```diff
@@ -12,6 +12,9 @@ import { Terminal } from './terminal'
 import { isSameUri, convertClientPathToDebugger, convertDebuggerPathToClient } from './paths'
 import minimatch = require('minimatch')
 
+let open = require('open')
```

Comment:
Please use `import open from 'open'` or this is not type safe. You'll need to add `npm install --save-dev @types/open`